src/transformState.js

```diff
@@ -62,7 +62,32 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  transforms.forEach(obj => {
```

Comment:
Що таке obj? Називай будь ласка змінну відповідно до її назначення

А можеш взагалі прибрати цей obj і зробити деструктуризацію ось таким чином
```suggestion
  transforms.forEach({ properties } => {
```
Тоді ти відразу отримаєш змінну properties, і надалі можеш писати її без приставки obj.